frontend/app/pages/tasks/description/[id].vue

```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+import { FetchError } from 'ofetch'
+
+const description = useState('task-description', () => '')
+
+const route = useRoute()
+const task_id = String(route.params.id ?? 'simple-task-example')
+
+if (import.meta.server) {
+  try {
+    const address = '/tasks/description/{task_id}'
+    const { data: response } = await useApi(address, {
+      path: { task_id },
+    })
+
+    description.value = response.value ?? 'No task here'
```

Comment:
Umm i don't like that. Move it below the `if` statement. Check if it is undefined (and return). There is no need for `??` because we are displaying error page for that